.evergreen.yml

```diff
@@ -150,7 +150,11 @@ functions:
       working_dir: spruce
       script: |
         export PATH=/opt/nodejs/node-v16.13.0-linux-x64/bin:$PATH
-        ./node_modules/.bin/cypress run --record --key ${cypress_record_key} --reporter junit
+          cd /tmp
+          curl -L -o 'my_file.zip' 'https://www.googleapis.com/download/storage/v1/b/chromium-browser-snapshots/o/Linux_x64%2F1002917%2Fchrome-linux.zip?generation=1652398617890755&alt=media'
```

Comment:
This should probably be an expansion. You can add it to project variables.

Reply:
I tried in this commit but didn't get the correct result. I made sure to update the variable name in the settings legacy ui page

Reply:
It doesn't look like the `run-cypress-test` command even ran in that commit. You may have pushed a new commit to it https://evergreen.mongodb.com/lobster/evergreen/task/spruce_ubuntu1604_e2e_test_patch_321db0bd4b1817ce52a0dbd9e335339ee210dc13_62debe82d6d80a73df880aea_22_07_25_16_02_11/0/task#bookmarks=0%2C733&f~=100~run-cypress-tests&l=1

But did in a subsequent [commit](https://spruce.mongodb.com/task/spruce_ubuntu1604_e2e_test_patch_6c09c39220d1a3d8444d8782696e5d5cba647be3_62debf24a4cf476a8835c10c_22_07_25_16_04_54/logs?execution=0) before you reverted. The tests seemed to run before the spot host was reaped.

Reply:
The subsequent commit passes because the evergreen.yml  [changes](https://spruce.mongodb.com/version/62debf24a4cf476a8835c10c/changes?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC)  does include the expansion. Here are two patches that fail due to the expansion 
[[47a196b](https://github.com/evergreen-ci/spruce/pull/1329/commits/47a196b4d0dad29aa0e6e5498f291e5e20bd13b1)](https://spruce.mongodb.com/task/spruce_ubuntu1604_e2e_test_patch_6c09c39220d1a3d8444d8782696e5d5cba647be3_62debf24a4cf476a8835c10c_22_07_25_16_04_54/logs?execution=1)
[[25cfb7a](https://github.com/evergreen-ci/spruce/pull/1329/commits/25cfb7a1ce12886a8b46efe0ee6e25bc4c2ba964)](https://spruce.mongodb.com/task/spruce_ubuntu1604_e2e_test_patch_312cde48481a5a199aadaf4c24a1946c45892418_62e19bc4d1fe0746b4d986db_22_07_27_20_10_46/logs?execution=0)

Reply:
The 2nd execution of the subsequent commit failed because the chrome download wasn't found. This happened again in this [patch](https://spruce.mongodb.com/task/spruce_ubuntu1604_e2e_test_patch_312cde48481a5a199aadaf4c24a1946c45892418_62e19bc4d1fe0746b4d986db_22_07_27_20_10_46/logs?execution=0) but the latest commit seems to be fine.